search_events_app/tests.py

```diff
@@ -44,6 +45,12 @@ def test_feature_model(self):
 
     def test_country_model(self):
         self.assertEqual(self.country.name, "Argentina")
+        self.assertEqual(self.country.alpha_2_code, "AR")
```

Comment:
Maybe you can separate the tests related to the Country model from the tests related to the Event Model. This could be more readable these tests...

Reply:
Do you mean to create a test class for each django model?